MobileBuy/buy/src/main/java/com/shopify/buy3/HttpCallbackWithRetry.java

```diff
@@ -125,8 +125,8 @@ private void notifyError(final GraphError error) {
     Runnable action = () -> {
       if (error instanceof GraphNetworkError) {
```

Comment:
Is it actually worthwhile to break this out by error type? It looks like all we end up doing is calling `callback.onFailure()` anyway, unless I'm missing something outside this PR